app/dependencies.php

```diff
@@ -24,5 +26,10 @@
 
             return $logger;
         },
+        SettingsInterface::class => function (ContainerInterface $c) {
+            $settings = $c->get('settings');
+
+            return new Settings($settings);
+        }
```

Comment:
This definition should actually go in the `settings.php` file and we should replace the `settings` definition with this one instead.

Reply:
```suggestion
return function (ContainerBuilder $containerBuilder) {
    // Global Settings Object
    $containerBuilder->addDefinitions([
        SettingsInterface::class => function () {
            $settings = [
                'displayErrorDetails' => true, // Should be set to false in production
                'logger' => [
                    'name' => 'slim-app',
                    'path' => isset($_ENV['docker']) ? 'php://stdout' : __DIR__ . '/../logs/app.log',
                    'level' => Logger::DEBUG,
                ],
            ];

            return new Settings($settings);
        },
    ]);
};
```

do you want to remove this definition and replace it the settings.php like this?